eslzArm/managementGroupTemplates/policyAssignments/DINE-SQLAuditingPolicyAssignment.json

```diff
@@ -19,7 +25,7 @@
     },
     "variables": {
         "policyDefinitions": {
-            "deploySqlAuditing": "/providers/Microsoft.Authorization/policyDefinitions/a6fb4358-5bf4-4ad7-ba82-2cd2f41ce5e9"
+            "deploySqlAuditing": "/providers/Microsoft.Authorization/policyDefinitions/25da7dfb-0666-4a15-a8f5-402127efd8bb"
         },
         "policyAssignmentNames": {
             "deploySqlAuditing": "Deploy-SQL-DB-Auditing",
```

Comment:
Think we need to give it a new assignment name, to make Terraform modules life easier. @krowlandson or @matt-FFFFFF can you confirm?

If so I propose `Deploy-AzSqlDb-Auditing`.

Also will need to update ALZ-Policies page in wiki (aka.ms/alz/policies) and whats new page with this assignment name change

Reply:
I will update ALZ-Policies.md, but I will wait for input on changing the overall assignment name

Reply:
ALZ-Policies.md updated; awaiting further input for potential assignment renaming

Reply:
@matt-FFFFFF or @krowlandson can you comment please on the assignment name and whether we should change it to make the TF modules life easier?

Reply:
Hi,

Yes please can we use a new assignment here. It will make the lifecycle easier to manage for state based deployments! Thanks

Reply:
Thanks @matt-FFFFFF. @pkorolo please change its name and rename the file etc. to match as per the suggestion I shared above.
`Deploy-AzSqlDb-Auditing`

Reply:
@jtracey93 this is just this _variable_ name change from "Deploy-SQL-DB-Auditing" to "Deploy-AzSqlDb-Auditing", right?
I do not see other places needing update because of this, not even the what's new tmho.

On the other hand, I have found an error of mine, which does have implications on the policy assignment file _itself_, as well as in what's new in wiki and the alz-policies, so I need to synch with you _before_ i submit any new changes.